src/client.ts

```diff
@@ -9325,6 +9331,19 @@ export class MatrixClient extends TypedEventEmitter<EmittedEvents, ClientEventHa
     }
 }
 
+export function getCount({ type, room, event }: {
```

Comment:
Could this be moved to somewhere more logical, `getCount` in `client.ts` implies it gets a count of clients/a client, which it doesn't